Dockerfile.QA

```diff
@@ -339,6 +339,11 @@ RUN pip3 install --upgrade wheel setuptools && \
     pip3 install --upgrade numpy pillow attrdict future grpcio requests gsutil \
                            awscli six grpcio-channelz prettytable virtualenv
 
+# L0_http_fuzz is hitting similar issue with boofuzz with latest version (0.4.0):
+# https://github.com/jtpereyda/boofuzz/issues/529
+# Hence, fixing the boofuzz version to 0.3.0
+RUN pip3 install 'boofuzz==0.3.0'
```

Comment:
It causing en issue because it's trying to install `blinker` version `1.6` which previously installed with `apt install software-properties-common`  package that has version `1.4`. Unfortunately `pip` utility unable to upgraded OS installed packages.